meeting-notes/2024/2024-03-12_europe_africa_cc.md

```diff
@@ -0,0 +1,29 @@
+# scikit-image Community Call
+
+- **Time:** Tuesday, March 12, 2024, 17:00 – 18:00 UTC
+- **[Join video call via Jitsi](https://meet.evolix.org/skimage-meeting)**
+- ~~**[Join video call via Zoom](https://us06web.zoom.us/j/88060567580?pwd=THRpaWFnSFNwK0Fycy9FVk5RYnV5UT09)**~~
+- **Notes:** https://hackmd.io/@scientific-python/HJWUBDVNi
+- **[scikit-image Community Calendar](https://scientific-python.org/calendars/skimage.ics)**
+- **[Archive — Meeting Notes](https://github.com/scikit-image/meeting-notes)**
+- **[Code of Conduct](https://scikit-image.org/docs/stable/conduct/code_of_conduct.html)**
+
+**Present:** Stéfan, Lars
+
+## Agenda
+
+- [CI jobs with optional dependencies are failing #7339](https://github.com/scikit-image/scikit-image/issues/7339)
+  - [Possible fix #7340](https://github.com/scikit-image/scikit-image/pull/7340)
+- Plugin's in `skimage.io`
+  - Is `imread` plugin still relevant? 
+    https://pypi.org/project/imread/ is no longer a dependency
+  - Same with `osgeo.gdal`
+  - I suspect nose style `setup` and `teardown` have broken for some time now ([deprecated by pytest](https://docs.pytest.org/en/latest/changelog.html#id179))
+    We need to clean that up ASAP
+  - For 2.0, remove plugin infrastructure, support thin interface around `imageio`; users should simply bring their own NumPy arrays. We can document libraries like fits, etc.
```

Comment:
Link/reference for `fits`: https://pypi.org/project/fits/ or https://pypi.org/project/xarray-fits/?

Reply:
[`astropy.io.fits`](https://docs.astropy.org/en/stable/io/fits/index.html) actually.

https://github.com/scikit-image/scikit-image/blob/5b732a026de26c7f8e5cc311075ba81a0d884d4a/skimage/io/_plugins/fits_plugin.py#L6

```suggestion
  - For 2.0, remove plugin infrastructure, support thin interface around `imageio`; users should simply bring their own NumPy arrays. We can document libraries like [fits](https://docs.astropy.org/en/stable/io/fits/index.html), etc.
```

Reply:
Perfect, thanks!